<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/DefaultplotDialog.ui,v $ </comment>
<comment>   $Revision: 1.2.40.2 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: ssahle $ </comment>
<comment>   $Date: 2011/10/24 15:28:11 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>DefaultPlotDialog</class>
 <widget class="QDialog" name="DefaultPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="listBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditTitle"/>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>show all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Create!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="global">vector</include>
  <include location="local">copasi.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DefaultPlotDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButton</sender>
   <signal>clicked()</signal>
   <receiver>DefaultPlotDialog</receiver>
   <slot>slotCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DefaultPlotDialog</receiver>
   <slot>slotToggleAll(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listBox</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>DefaultPlotDialog</receiver>
   <slot>slotSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotToggleAll(bool)</slot>
  <slot>slotSelect()</slot>
 </slots>
</ui>
